#!/usr/bin/env python
PACKAGE = "generic_potential_field"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()


# max distance to feel the obstacle potential
gen.add("tPotFuncZeroD",  double_t, 0, "tPotFuncZeroD", 0.5,  0, 5)


# min distance to feel the obstacle potential
gen.add("tPotFuncInfD", double_t, 0, "tPotFuncInfD",0.2,  0, 5)

# max value of the potential function
gen.add("tPotFuncSatValue", double_t, 0, "tPotFuncSatValue",4.0,  0, 15)


# gain to scale the potential value
gen.add("tPotFuncGain",  double_t, 0, "tPotFuncGain", 100,  0, 500)

exit(gen.generate(PACKAGE, "generic_potential_field", "potentialParams")) 

